libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -19,27 +20,32 @@ import {
     ViewEncapsulation
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
-import { DOWN_ARROW, TAB, SPACE, ENTER } from '@angular/cdk/keycodes';
+import { DOWN_ARROW, TAB, SPACE, ENTER, ESCAPE } from '@angular/cdk/keycodes';
+import { LiveAnnouncer } from '@angular/cdk/a11y';
 import { Subscription } from 'rxjs';
-import { PopoverComponent } from '@fundamental-ngx/core/popover';
-import { MenuKeyboardService } from '@fundamental-ngx/core/menu';
-import { FormStates, PopoverFillMode } from '@fundamental-ngx/core/shared';
-import { MobileModeConfig } from '@fundamental-ngx/core/mobile-mode';
-import { TokenizerComponent } from '@fundamental-ngx/core/token';
-import { ListComponent } from '@fundamental-ngx/core/list';
+
 import {
+    applyCssClass,
     ContentDensityService,
     CssClassBuilder,
     DynamicComponentService,
-    RtlService,
-    applyCssClass,
     FocusEscapeDirection,
     KeyUtil,
+    RtlService,
     FocusTrapService
 } from '@fundamental-ngx/core/utils';
+import { ListComponent } from '@fundamental-ngx/core/list';
+import { MenuKeyboardService } from '@fundamental-ngx/core/menu';
+import { MobileModeConfig } from '@fundamental-ngx/core/mobile-mode';
+import { PopoverComponent } from '@fundamental-ngx/core/popover';
+import { FormStates, PopoverFillMode } from '@fundamental-ngx/core/shared';
+import { TokenizerComponent } from '@fundamental-ngx/core/token';
+
 import { MultiInputMobileComponent } from './multi-input-mobile/multi-input-mobile.component';
 import { MULTI_INPUT_COMPONENT, MultiInputInterface } from './multi-input.interface';
 
+let inputRandomId = 0;
```

Comment:
It's actually not `random` but `unique`. Please be consistent with the other components in the library: 
<img width="317" alt="Screen Shot 2021-08-09 at 2 37 33 PM" src="https://user-images.githubusercontent.com/39598672/128756776-7715a5fa-0a8d-4a47-9ffa-3f2f695e2058.png">

Reply:
Done